extensions/amp-accordion/1.0/README.md

```diff
@@ -0,0 +1,641 @@
+
+```javascript
+ref.current.expand();
+ref.current.expand('section1');
+```
+
+**collapse()**
+The `collapse` action collapses the sections of the `bento-accordion`. If a
+section is already collapsed, it stays collapsed. When called with no arguments,
+it collapses all sections of the accordion. To specify a section, add the
+`section` argument, and use its corresponding `id` as the value.
+
+```javascript
+ref.current.collapse();
+ref.current.collapse('section1');
+```
+
+##### Events
+
+The Bento Accordion API allows you to respond to the following events:
+
+**onExpandStateChange**
+This event is triggered on a section when an accordion section is expanded or collpased and is dispatched from the expanded section.
+
+See below for example.
+
+**onCollapse**
+This event is triggered on a section when an accordion section is collapsed and is dispatched from the collapsed section.
+
+In the example below, `section 1` listens for the `expand` event and expands `section 2` when it is expanded. `section 2` listens for the `collapse` event and collapses `section 1` when it is collapsed.
+
+See below for example.
+
+```jsx
+    <BentoAccordion ref={ref}>
+      <BentoAccordionSection
+        id="section1"
+        key={1}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section1 expanded') : alert('section1 collapsed')
+        }>
+        <BentoAccordionHeader><h1>Section 1</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 1</BentoAccordionContent>
+      </BentoAccordionSection>
+
+      <BentoAccordionSection
+        id="section2"
+        key={2}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section2 expanded') : alert('section2 collapsed'}
+        }>
+        <BentoAccordionHeader><h1>Section 2</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 2</BentoAccordionContent>
+      </BentoAccordionSection>
+
+      <BentoAccordionSection
+        id="section3"
+        key={3}
+        onExpandStateChange={
+          (expanded) => expanded ? alert('section3 expanded') : alert('section3 collapsed'}
+        }>
+        <BentoAccordionHeader><h1>Section 3</h1></BentoAccordionHeader>
+        <BentoAccordionContent>Content 3</BentoAccordionContent>
+      </BentoAccordionSection>
+    </BentoAccordion>
+```
+
+#### Layout and style
+
+**Container type**
+
+The `BentoAccordion` component has a defined layout size type. To ensure the component renders correctly, be sure to apply a size to the component and its immediate children via a desired CSS layout (such as one defined with `height`, `width`, `aspect-ratio`, or other such properties). These can be applied inline:
+
+```jsx
+<BentoAccordion style={{width: '300px', height: '100px'}}>
+  ...
+</BentoAccordion>
+```
+
+Or via `className`:
+
+```jsx
+<BentoAccordion className='custom-styles'>
+  ...
+</BentoAccordion>
+```
+
+```css
+.custom-styles {
+  height: 100px;
+  width: 100%;
+}
```

Comment:
Let's add more likely styles than dimensions for this example, perhaps a border or bg color.

Reply:
added bg color